registration/serializers.py

```diff
@@ -44,11 +53,38 @@ class Meta:
             "address",
             "preferred_comms",
             "num_children",
+            "enrolled",
+            "current_class",
+            "status",
         ]
 
     def get_num_children(self, obj):
         return obj.children.count()
 
+    def get_enrolled(self, obj):
+        most_recent_enrolment = Enrolment.objects.filter(
+            family=obj, session=self.most_recent_session
+        )
```

Comment:
I have this logic in all 3 methods (mainly bc of the family=obj line), is there a better approach for this? :o